### DISCUSSION TITLE
Wurst Client 1.17: ModResolutionException: Mod discovery failed!

### DISCUSSION THREAD

**Discussion by imlens** [FabricMC/fabric-api#2890] [Player support]
Hello! I'm using Wurst Client for 1.17 and I followed the tutorial [here](https://www.wurstclient.net/tutorials/how-to-install/wurst-7/). Everything seems normal and the fabric loader appears in the Minecraft launcher. After I launch, Fabric loader opens with an error as follows

![image](https://user-images.githubusercontent.com/83427897/216912354-58fa9b32-0f1c-4a22-af56-0daf480d6d65.png)

I have no clue what's happening. I checked the bottom of the Wurst tutorial page, but it doesn't show any solutions to my problem
Thanks beforehand for any help!

**Comment by imlens** [comment]
# Update
After moving the mods I had into another folder, I'm getting this error:
![image](https://user-images.githubusercontent.com/83427897/216978936-c466d88e-75cd-4bf1-b287-e68fdec18e66.png)

**Comment by Alexander01998** [comment]
Hello 1enss,

The first error you are getting is caused by a corrupted ZIP file called `cloud-1.8.jar`, which isn't supposed to be there. It seems like you have already figured out that this file needs to be removed from your mods folder.

The second error is caused by you selecting 1.17 in the Fabric installer, but using the 1.17.1 versions of all of your mods (Wurst and Fabric API). To fix this, either select 1.17.1 in the Fabric installer or replace all of your 1.17.1 mods with the 1.17 versions, depending on which Minecraft version you want to use.

As for the tutorial page, I would like to point out that the second error is actually explained in this section: https://www.wurstclient.net/tutorials/how-to-install/wurst-7/#error-replace-minecraft-x-with-version-y

I hope this helps. If you have any more questions, feel free to reach out to me. However, I must warn you that posting about this in the Fabric discussion board is likely not a good idea. From my experience, the moderators here do not take kindly to people using hacked clients. You are better off asking in one of the Wurst Client communities, such as the [Wurst7 issue tracker](https://github.com/Wurst-Imperium/Wurst7) or the [r/WurstClient subreddit](https://www.reddit.com/r/WurstClient/).

-Alexander01998
